make_html.py

```diff
@@ -0,0 +1,57 @@
+import os
+import yaml
+import shutil
+import jinja2
+
+
+def get_options():
+    import argparse
+    parser = argparse.ArgumentParser(
+        description="Get starcheck checks")
+    parser.set_defaults()
+    parser.add_argument("--infile",
+                        default="checks.yaml",
+                        help="file of checks (output from previous run of tool)")
+    parser.add_argument("--outdir",
+                        default="report_html")
+    args = parser.parse_args()
+    return args
+
+jinja_env = jinja2.Environment(
+    loader=jinja2.FileSystemLoader('templates'))
+jinja_env.line_comment_prefix = '##'
+jinja_env.line_statement_prefix = '#'
+
+
+opt = get_options()
+if not os.path.exists(opt.outdir):
+    os.makedirs(opt.outdir)
+shutil.copy("prism_ocadia.css", opt.outdir)
+shutil.copy("prism.js", opt.outdir)
+shutil.copy("sorttable.js", opt.outdir)
+
+checks = yaml.load(open(opt.infile).read())
+
+displaycols = ['id', 'link', 'filename', 'line_number',
```

Comment:
`displaycols` can be (in order):
- `title` which is linked to the full entry
- `file` which is in the form `filename : line_number` and is linked to the `github_link`
- `type`
- `severity`
- `aca_checklist`

If we ever have meaningful content in `hopper` that might be added as a column.